ports/unix/mbedtls/mbedtls_config_port.h

```diff
@@ -32,7 +32,18 @@
 // Enable mbedtls modules
 #define MBEDTLS_TIMING_C
 
+#if MICROPY_UNIX_COVERAGE
```

Comment:
Existing uses of this feature macro are `#if defined(MICROPY_UNIX_COVERAGE)`.

I know it's probably equivalent, but this macro is not defined (even to 0) when coverage is not being built, and probably good to keep consistency with existing uses of it.

Reply:
Have changed.

Would you be open to a PR that cleaned up the few `MICROPY_` boolean flags that are treated as ("defined means set" and changed them to match the pattern more commonly used in MicroPython of "1 means set, 0 means unset")?

Quick grep suggests there are not many (pattern is mostly used when the macro holds a value like a pin number).

Reply:
> Would you be open to a PR that cleaned up the few `MICROPY_` boolean flags that are treated as ("defined means set" and changed them to match the pattern more commonly used in MicroPython of "1 means set, 0 means unset")?

Yes!